crates/solver/src/settlement_submission/flashbots_settlement.rs

```diff
@@ -202,7 +202,7 @@ impl<'a> FlashbotsSolutionSubmitter<'a> {
         transactions: &mut Vec<H256>,
     ) -> anyhow::Error {
         const UPDATE_INTERVAL: Duration = Duration::from_secs(5);
-        const MINIMAL_MAX_PRIORITY_FEE: f64 = 10_000_000_000.0;
+        const MINIMAL_MAX_PRIORITY_FEE: f64 = 3_010_000_000.0;
```

Comment:
Would it be possible to make this configurable? We can even configure it to be an additional Flashbot tip in GWei:
```
--additional-flashbot-tip 3.0
```

Reply:
I think this would be good. So we don't have to rebuild if we see TXs not getting mined because of this.

Reply:
Sure, added.